tests/e2e/run-e2e-test.sh

```diff
@@ -35,6 +35,8 @@ export SHAREDNFS=false
 
 export INSTALL_VAULT=false
 
+export PROXY_HOST="csm-authorization.com"
```

Comment:
This requires that `/etc/hosts` is configured with this hostname and the bastion/master node of the cluster. Is the README updated with that?